README.md

```diff
@@ -17,4 +17,28 @@ export PYTHONPATH="$(pwd)"
 ## How to test?
 ```
 pytest
-```
\ No newline at end of file
+```
+
+## Work flow
+1. checkout branch DQA-<ticket_number>
```

Comment:
Maybe start with the capital letters?

Reply:
do you mean 
1. Checkout branch
2. Coding
3. Fetch ....
....
right?

Reply:
Yeah, maybe will look more professional XD

Reply:
Updated

Reply:
@dang113108 updated again